plugin-cgroupv2/src/cgroupv2.rs

```diff
@@ -24,16 +24,30 @@ pub struct CgroupMeasurements {
     pub cpu_time_user_mode: u64,
     /// CPU in system mode usage time by the cgroup.
     pub cpu_time_system_mode: u64,
+    /// Resident memory usage (RSS) currently used by the cgroup.
+    pub memory_usage_resident: u64,
     /// Anonymous used memory, corresponding to running process and various allocated memory.
     pub memory_anonymous: u64,
-    // Files memory, corresponding to open files and descriptors.
+    /// Files memory, corresponding to open files and descriptors.
     pub memory_file: u64,
-    // Memory reserved for kernel operations.
+    /// Memory reserved for kernel operations.
     pub memory_kernel: u64,
     /// Memory used to manage correspondence between virtual and physical addresses.
     pub memory_pagetables: u64,
 }
 
+impl CgroupMeasurements {
+    pub fn load_memory_current_from_str(&mut self, s: &str) -> Result<()> {
+        match s.trim().parse::<u64>() {
+            Ok(value) => self.memory_usage_resident = value,
+            Err(e) => {
+                eprintln!("Failed to parse '{}': {}", s, e);
```

Comment:
You should return an error instead of printing to stderr here :p

Reply:
resolved !